front/src/applications/operationalStudies/views/ManageTrainSchedule.tsx

```diff
@@ -119,7 +119,7 @@ const ManageTrainSchedule = ({ trainIdToEdit }: ManageTrainScheduleProps) => {
         <div className="floating-itinerary">
           <Itinerary pathProperties={pathProperties} setPathProperties={setPathProperties} />
         </div>
-        <Map pathProperties={pathProperties}>
+        <Map pathProperties={pathProperties} pathGeometry={pathProperties.geometry}>
```

Comment:
```suggestion
        <Map pathProperties={pathProperties} pathGeometry={pathProperties?.geometry}>
```
`pathProperties` is possibly `undefined` here :)